### PR TITLE
Impostor Radio & PushToTalk Compatability

### PR DESCRIPTION
**Pull request by JKohlman**
I don't actually have the setup to test this at the moment but I believe currently radio & push to talk requires two buttons at once. This should make it so that just holding the radio button works.

### CODE REVIEW

File: src/main/hook.ts [hunk 1]
```diff
@@ -82,6 +83,7 @@ ipcMain.handle(IpcHandlerMessages.START_HOOK, async (event) => {
 				event.sender.send(IpcRendererMessages.TOGGLE_MUTE);
 			}
 			if (keyCodeMatches(impostorRadioShortcut!, keyId)) {
+				event.sender.send(IpcRendererMessages.PUSH_TO_TALK, false);
```
Comment (OhMyGuus) [inline src/main/hook.ts]: What if the player still has his push to talk button pressed

**Review by OhMyGuus — CHANGES_REQUESTED** [review]

File: src/main/hook.ts (hunk 1 above)
Comment (OhMyGuus) [reply src/main/hook.ts]: Apparently this comment was still on pending thought you where able to see it when it was on pending

Comment (JKohlman) [reply src/main/hook.ts]: Should be fixed by latest commit

**Comment by JKohlman** [comment]
I resolved those changes, not sure why it's still saying changes requested.

**Pull request merged**